Update where-to.json

cnjoyce1225 opened the pull request:
Added the location property and fixed the comma inconsistency

cnjoyce1225 committed:
Update where-to.json

Added the location property and fixed the comma inconsistency

wfish-ghoti commented:
Hello @cnjoyce1225!

Before I approve this for a merge, could you please test it by running the server? (I added the files to do this to the api folder last night.) 

Wendy


```
cd <your-github-workspace>/where-to-2024/api 
sh start-server.sh
```

cnjoyce1225 commented:
Hey @wfish-ghoti ! I am really stuck and have genuinely no clue how to test this 😣 I'm going to try and meet with Bob tomorrow to see work this out.

cnjoyce1225 committed:
Update where-to.json

I removed the extra commas! Bob said he tested it without them and he said it worked.
